openfoodfacts_exports/exports/parquet.py

```diff
@@ -212,4 +297,67 @@ def push_parquet_file_to_hf(
         path_in_repo="products.parquet",
         commit_message=commit_message,
     )
-    logger.info(f"Data succesfully pushed to Hugging Face at {repo_id}")
+    logger.info("Data succesfully pushed to Hugging Face at %s", repo_id)
+
+
+@timer
```

Comment:
Thank you for adding a timer wrapper, it's useful for debug. For prod I think we can drop it here, we can know how long it takes by looking at the INFO log messages at the beginnning and end of `postprocess_parquet`.